torch/_inductor/fx_passes/quantization.py

```diff
@@ -146,7 +167,12 @@ def generate_pattern_with_output_quant(computation_call):
                         aten.round.default,
                         CallFunction(
                             aten.mul.Tensor,
-                            computation_call,
+                            computation_call
+                            if dtype == torch.float32
+                            else _generate_pattern_with_dtype_convert(
+                                computation_call,
+                                KeywordArg("autocast_output_quant_dtype"),
+                            ),
```

Comment:
looks like this appears multiple times, refactoring a bit would be better I think

Reply:
Same as above and changed accordingly.